lib/Cpdf.php

```diff
@@ -2975,6 +3076,136 @@ protected function o_embedded_file($id, $action, $options = null): ?string
         return null;
     }
 
+    /**
+     * Enable PDF/A compliance mode
+     */
+    public function enablePdfACompliance()
+    {
+        $this->pdfa = true;
+
+        $iccProfilePath = __DIR__ . '/res/sRGB2014.icc';
+        $this->o_catalog($this->catalogId, 'outputIntents', [
+            'iccProfileData' => file_get_contents($iccProfilePath),
+            'iccProfileName' => basename($iccProfilePath),
+            'colorComponentsCount' => '3',
+        ]);
+    }
+
+    /**
+     * Generate the Metadata XMP XML for PDF/A
+     *
+     * @return string
+     */
+    function getXmpMetadata()
+    {
+        $md = <<<EOT
+<?xpacket begin="ï»¿" id="W5M0MpCehiHzreSzNTczkc9d"?>
```

Comment:
If someone changes the file encoding it may break
Can this seqence be converted to "\xEF\xBB\xBF"
I am unsure my hex is good

Reply:
I believe that's right. Per the XMP spec the embedded XML "must be encoded as UTF-8." The document also handily indicates the BOM byte sequence to use for that encoding (0xEF 0xBB 0xBF).